content/docs/command-reference/version.md

```diff
@@ -36,36 +34,36 @@ usage: dvc version [-h] [-q | -v]
 
 The detail of DVC version depends upon the way of installing DVC.
 
-- **Official release**: [These instructions](/doc/install) include ways to
-  install DVC using the official package stored in Python Packaging Authority.
-  We mark these official releases with tags on DVC's repository. Any issues
-  reported with the official build can be traced using the `_BASE_VERSION`
-  constant
-  [in our core repo](https://github.com/iterative/dvc/blob/master/dvc/version.py).
-  For example `0.40.2`.
+- **Official release**: official releases (for example `1.7.2`) are stable
+  versions that have upgraded features and bug fixes.
+  [These instructions](/doc/install) include ways to install DVC using the
+  official package stored in [Python Packaging Authority](https://www.pypa.io/).
+  We mark these official releases with
+  [tags](https://github.com/iterative/dvc/releases) on DVC's repository. Any
+  issues reported with the official build can be traced using the
+  `_BASE_VERSION` constant in our
+  [core repo](https://github.com/iterative/dvc/blob/master/dvc/version.py).
 
   > 💡 To only get this number, use `dvc -V`.
 
 - **Development version**: `pip install git+git://github.com/iterative/dvc` will
-  install DVC using the `master` branch of DVC's repository. Another way of
-  setting up the development version is to clone the repository and run
-  `pip install .`.
+  install DVC using the `master` branch of DVC's repository. Please see
+  [development version](/doc/user-guide/contributing/core#development-environment),
+  another way of setting it up.
```

Comment:
Hmmm. Again, should there be a pip install... code block that doesn't correspond to the instructions in https://dvc.org/doc/user-guide/contributing/core#development-environment?

>another way of setting it up.

That's the problem. We probably shouldn't have 2... Let's double check which one to keep but most likely the one in the contrib guide only (so just the link is needed). Let's bring this up in Discord for confirmation though, please.

Reply:
Yea I think if both aren't needed. The link to contribution guide is better since in that method user can also make commits. Contribute basically.
> Let's bring this up in Discord for confirmation

Should I put this up on #Dev-Docs channel for confirmation?

Reply:
I see you removed it for now but yes, let's ask on Discord. #q-and-a is better in this case. You can link to the currently published dvc version ref and ask why is that pip install different than the instructions in the docs contrib guide, which one is better, are both useful in different scenarios, etc. Thanks

Reply:
I see you asked in https://discord.com/channels/485586884165107732/563406153334128681/764562383825272872, thanks. That's not the question we're trying to answer though. I re-asked in https://discord.com/channels/485586884165107732/563406153334128681/764910839987699752 — let's hope we get an answer before this is ready to merge...

Reply:
I guess not 😅 — well, we can always come back to this on a separate PR.

Reply:
OK I pinged the team internally and here's the answer:

> Hmm, as far as I know,
`pip install git+git://github.com/iterative/dvc`
is just a convenience method for git clone && cd && pip install .
To do our development installation, we could update it to look as follows: `pip install git+git://github.com/iterative/dvc#egg=dvc[all,tests]`
But to be honest, its just git clone and pip install packed into a single command. Since it does pretty much the same, I would say we can get rid of that - in case of development installation, we don't have to provide every possible way to do that.

So it looks like it's OK to just remove it as we've done.